test/conformance/chainsaw/webhooks/pod-all-subresources/webhooks.yaml

```diff
@@ -17,12 +17,101 @@ webhooks:
     - UPDATE
     resources:
     - pods/attach
+    scope: 'Namespaced'
+  - apiGroups:
+    - ""
+    apiVersions:
+    - v1
+    operations:
+    - CONNECT
+    - CREATE
+    - DELETE
+    - UPDATE
+    resources:
     - pods/binding
+    scope: 'Namespaced'
+  - apiGroups:
+    - ""
+    apiVersions:
+    - v1
+    operations:
+    - CONNECT
+    - CREATE
+    - DELETE
+    - UPDATE
+    resources:
     - pods/ephemeralcontainers
+    scope: 'Namespaced'
+  - apiGroups:
+    - ""
+    apiVersions:
+    - v1
+    operations:
+    - CONNECT
+    - CREATE
+    - DELETE
+    - UPDATE
+    resources:
     - pods/eviction
+    scope: 'Namespaced'
+  - apiGroups:
+    - ""
+    apiVersions:
+    - v1
+    operations:
+    - CONNECT
+    - CREATE
+    - DELETE
+    - UPDATE
+    resources:
     - pods/exec
+    scope: 'Namespaced'
+  - apiGroups:
+    - ""
+    apiVersions:
+    - v1
+    operations:
+    - CONNECT
+    - CREATE
+    - DELETE
+    - UPDATE
+    resources:
     - pods/log
+    scope: 'Namespaced'
+  - apiGroups:
+    - ""
+    apiVersions:
+    - v1
+    operations:
+    - CONNECT
+    - CREATE
+    - DELETE
+    - UPDATE
+    resources:
     - pods/portforward
+    scope: 'Namespaced'
```

Comment:
Can we just append the resource if the operations and the scope are the same?

Instead of the following:
```
  - apiGroups:
    - ""
    apiVersions:
    - v1
    operations:
    - CONNECT
    - CREATE
    - DELETE
    - UPDATE
    resources:
    - pods/log
    scope: 'Namespaced'
  - apiGroups:
    - ""
    apiVersions:
    - v1
    operations:
    - CONNECT
    - CREATE
    - DELETE
    - UPDATE
    resources:
    - pods/portforward
    scope: 'Namespaced'
```

It is better to combine them since operations, scope, API version and group are the same.
```
  - apiGroups:
    - ""
    apiVersions:
    - v1
    operations:
    - CONNECT
    - CREATE
    - DELETE
    - UPDATE
    resources:
    - pods/log
    - pods/portforward
    scope: 'Namespaced'
```

Reply:
Agreed, I will add these changes.